_security/access-control/permissions.md

```diff
@@ -65,6 +65,16 @@ Rather than individual permissions, you can often achieve your desired security
 {: .tip }
 
 
+### System permission
+
+The system permission `system:admin/system_index` is unique among other permissions in that it extends some traditional admin-only accessibility to non-admin users. This permission gives normal users the ability to modify system indexes for the cluster. It excludes, however, access to the security system index `.opendistro_security`, which is used to store the configuration YAML files and remains accessible only to admins with an admin certificate.
```

Comment:
This sounds like the permission itself is `system:admin/system_index` however `system_index` here is actually a placeholder text that should be replaced by the actual name of the system index i.e. `system:admin/.opendistro-anomaly-detectors`